### ISSUE TITLE
check to see if the 'skip to transcript doesn't work if jaws running' bug is present

### ISSUE THREAD

**Issue by moloko** [adaptlearning/adapt-vimeo#3]
the main adapt media component had [a bug where the skip to transcript feature didn't work if JAWS was running](https://github.com/adaptlearning/adapt_framework/issues/2427), suspect that may also exist here, needs checking

**Comment by zarek3333** [comment]
Hey there I was testing with JAWS as we use it alot here too. You have a couple errors in the way in which it works in accessible mode. That said everything else looks amazing, nice work on this component! Below are the issues I ran across and the fixes I used:

1.) In TranscriptView.js on line 41 : 
var $buttonText = this.$('.**youtube**__transcript-btn-text');
should be:
var $buttonText = this.$('.**vimeo**__transcript-btn-text');

**_This causes the Close transcript label not to show without this fix._**

2.) In Jaws on triggering the Open Transcript button it doesn't highlight the Transcript body text. To fix this in TranscriptView.js on  lines 52-54 I replaced with:
$transcriptBodyContainer.stop(true, true)[slide](function() {
        Adapt.trigger('device:resize');
      })**.a11y_focus();**

Here is my fixed forked accessible version...[https://github.com/zarek3333/adapt-vimeo](https://github.com/zarek3333/adapt-vimeo)

**Comment by moloko** [comment]
@zarek3333 thanks for the detailed issue! I’ll take a look at this tomorrow

**Comment by oliverfoster** [comment]
fixed with https://github.com/adaptlearning/adapt-contrib-core/pull/90
